ios/browser/api/ai_chat/DEPS

```diff
@@ -0,0 +1,4 @@
+include_rules = [
+  "+ios/chrome/common/channel_info.h",
+  "+services/data_decoder/public/cpp",
+]
```

Comment:
you might want to configure your editor to automatically add a newline at the end so you don't get these